bucket/kdiff3.json

```
@@ -22,7 +22,7 @@
     ],
     "checkver": {
         "url": "https://download.kde.org/stable/kdiff3/?C=M;O=D",
-        "regex": "(?<file>kdiff3-((?<version>[\\d.]+))-windows[-_]64(-cl)?)\\.exe"
+        "regex": "(?<file>kdiff3-((?<version>[\\d.\\w]+))-windows[-_]64(-cl)?)\\.exe"
```

```suggestion
        "regex": "(?<file>kdiff3-((?<version>[\\d.]+\\w?))-windows[-_]64(-cl)?)\\.exe"
```

Just an optional `\w` at the end of the version number.